Modules/Microsoft365DSC/DSCResources/MSFT_EXOAuthenticationPolicy/MSFT_EXOAuthenticationPolicy.schema.mof

```diff
@@ -0,0 +1,27 @@
+[ClassVersion("1.0.0.0"), FriendlyName("EXOAuthenticationPolicy")]
+class MSFT_EXOAuthenticationPolicy : OMI_BaseResource
+{
+    [Key, Description("The Identity parameter specifies the authentication policy you want to view or modify.")] String Identity;
+    [write, Description("The AllowBasicAuthActiveSync switch specifies whether to allow Basic authentication with Exchange Active Sync")] Switch AllowBasicAuthActiveSync;
+    [write, Description("The AllowBasicAuthAutodiscover switch specifies whether to allow Basic authentication with Autodiscover")] Switch AllowBasicAuthAutodiscover;
+    [write, Description("The AllowBasicAuthImap switch specifies whether to allow Basic authentication with IMAP.")] Switch AllowBasicAuthImap;
+    [write, Description("The AllowBasicAuthMapi switch specifies whether to allow Basic authentication with MAPI.")] Switch AllowBasicAuthMapi;
+    [write, Description("The AllowBasicAuthOfflineAddressBook switch specifies whether to allow Basic authentication with Offline Address Books.")] Switch AllowBasicAuthOfflineAdressBook;
+    [write, Description("The AllowBasicAuthOutlookService switch specifies whether to allow Basic authentication with the Outlook service.")] Switch AllowBasicAuthOutlookService;
+    [write, Description("The AllowBasicAuthPop switch specifies whether to allow Basic authentication with POP.")] Switch AllowBasicAuthPop;
+    [write, Description("The AllowBasicAuthPowerShell switch specifies whether to allow Basic authentication with PowerShell.")] Switch AllowBasicAuthPowershell;
+    [write, Description("The AllowBasicAuthReporting Web Services switch specifies whether to allow Basic authentication with reporting web services.")] Switch AllowBasicAuthReportingWebServices;
+    [write, Description("The AllowBasicAuthRpc switch specifies whether to allow Basic authentication with RPC.")] Switch AllowBasicAuthRpc;
+    [write, Description("The AllowBasicAuthSmtp switch specifies whether to allow Basic authentication with SMTP.")] Switch AllowBasicAuthSmtp;
+    [write, Description("The AllowBasicAuthWebServices switch specifies whether to allow Basic authentication with Exchange Web Services (EWS).")] Switch AllowBasicAuthWebServices;
+    [Write, Description("The AccessRight parameter specifies the permission that you want to assign in the application access policy."), ValueMap{"RestrictAccess","DenyAccess"}, Values{"RestrictAccess","DenyAccess"}] String AccessRight;
+    [Write, Description("The AppID parameter specifies the GUID of the apps to include in the policy.")] String AppID[];
+    [Write, Description("The PolicyScopeGroupID parameter specifies the recipient to define in the policy. You can use any value that uniquely identifies the recipient.")] String PolicyScopeGroupId;
+    [Write, Description("Specify if the Application Access Policy should exist or not."), ValueMap{"Present","Absent"}, Values{"Present","Absent"}] String Ensure;
```

Comment:
Make sure you update the description to reflect the Auth Policy

Reply:
Updated descriptions to reflect correct module.